bigtop_toolchain/manifests/packages.pp

```diff
@@ -34,7 +34,6 @@
         "fuse-devel",
         "cppunit-devel",
         "openssl-devel",
-        "python2-pip",
```

Comment:
We do not python2-pip for CentOS 7 and CentOS 8?

Reply:
I think we can drop pip2 from CentOS 7 and 8 too, for the same reason as #728.